src/autopas/selectors/AutoTuner.h

```diff
@@ -320,10 +320,9 @@ void AutoTuner<Particle, ParticleCell>::iteratePairwiseTemplateHelper(PairwiseFu
   // if tuning execute with time measurements
   if (inTuningPhase) {
     auto start = std::chrono::high_resolution_clock::now();
-    // @todo remove useNewton3 in iteratePairwise by introducing traversals for DS and VL
 
     f->initTraversal();
-    withStaticContainerType(container, [&](auto container) { container->iteratePairwise(f, traversal.get()); });
+    container->iteratePairwise(traversal.get());
```

Comment:
I think if we do it this way `withStaticContainerType()` wouldn't be needed anymore correct?

Reply:
Correct! Do you want me to delete it?

Reply:
yes